client/i18n/locales/japanese/meta-tags.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
These can also be translated - we can work with Quincy and Ahmad to identify good SEO terms as well.

Reply:
I translated the title and some keywords which are often written in Japanese.